cmd/geoipupdate/config_test.go

```diff
@@ -216,10 +214,40 @@ EditionID GeoIP2-City
 			Err: "unknown option on line 2",
 		},
 		{
-			Description: "Missing required key",
-			Input: `Host updates.maxmind.com
+			Description: "Missing required key in options",
+			Input:       ``,
+			Err:         "the `EditionIDs` option is required",
+		},
+		{
+			Description: "LicenseKey is found but AccountID is not",
+			Input: `LicenseKey abcd
+EditionIDs GeoIP2-City
 `,
-			Err: "the `AccountID' option is required",
+			Err: "the `AccountID` option is required if the `LicenseKey` option is set",
+		},
+		{
+			Description: "AccountID 0 with the LicenseKey 000000000000 is treated as no AccountID/LicenseKey",
+			Input: `AccountID 0
+LicenseKey 000000000000
+EditionIDs GeoIP2-City`,
+			Output: &Config{
+				DatabaseDirectory: "/tmp",
+				EditionIDs:        []string{"GeoIP2-City"},
+				LockFile:          "/tmp/.geoipupdate.lock",
+				URL:               "https://updates.maxmind.com",
+			},
+		},
+		{
+			Description: "AccountID 999999 with the LicenseKey 000000000000 is treated as no AccountID/LicenseKey",
+			Input: `AccountID 999999
```

Comment:
Maybe it would be worthwhile adding a test for `999999` with some other license key.